pkg/config/consortiums.go

```diff
@@ -49,6 +49,42 @@ func (c *ConfigTx) AddOrgToConsortium(org Organization, consortium string) error
 	return nil
 }
 
+// GetConsortiums returns a list of consortiums for the channel configuration.
+// Consortiums is only defined for the ordering system channel.
+func (c *ConfigTx) GetConsortiums() ([]Consortium, error) {
```

Comment:
Going forward we don't need the `GET` prefix for these exported ConfigTX retrieval functions

Reply:
Sounds like there's an interesting story there. Were prefixes required previously?

Reply:
Interesting? Possibly but we have a [FAB-17662](https://jira.hyperledger.org/browse/FAB-17662) as a housekeeping item.

Reply:
done. will address rest of `GET` prefix in FAB-17662